docs/Manual/Programs/Starter/Options.md

```diff
@@ -274,10 +295,45 @@ This indicates whether or not a coordinator instance should be started
 This indicates whether or not a DB server instance should be started
 (default true).
 
-- `--server.rr=path`
+### Docker options
```

Comment:
Consider adding a like below like "The following options are applicable when running the _Starter_ in docker."